README.md

```diff
@@ -272,6 +273,17 @@ Alternatively you can pull from [Docker Hub](https://hub.docker.com/r/kuchy/pack
 docker pull kuchy/packtpub-crawler
 ```
 
+### cronjob
+Add this to your crontab to run the job daily at 9:
+```
+00 09 * * * /usr/bin/python PATH_TO_CRAWLER/script/spider.py --config PATH_TO_CRAWLER/config/prod.cfg --all --extras --notify join >> /tmp/packtpub.log 2>&1
+```
+
+### free ebook from the newsletter
+The script also downloads the free ebook from the weekly newsletter. The url (https://goo.gl/kUciut) is generated by an Google apps script which parses the mail. If you want to use your own source, just adapt url.bookFromNewsletter in the config. The url should point to an csv containing a single line with the url (no semicolon).
+You can also [clone the script](https://docs.google.com/spreadsheets/d/1jN5gV45uVkE0EEF4Nb-yVNfIr3o8OoiVveUZJRMiLFw) to use your own Gmail account. Just subscribe to the newsletter (found on the bottom of https://www.packtpub.com) and create a filter to tag your mails accordingly.
+
```

Comment:
That's ok, as I asked yesterday, is the google app something that you have done or where it comes from? Do you have the source code? I'd really like to see it. Is something that can be added to this project?